appinfo/routes.php

```diff
@@ -18,6 +18,7 @@
 		['name' => 'api1#index', 'url' => '/api/1/tables', 'verb' => 'GET'],
 		['name' => 'api1#createTable',	'url' => '/api/1/tables', 'verb' => 'POST'],
 		['name' => 'api1#updateTable',	'url' => '/api/1/tables/{tableId}', 'verb' => 'PUT'],
+		['name' => 'api1#transferTable',	'url' => '/api/1/tables/{tableId}', 'verb' => 'PUT'],
```

Comment:
I don't understand how routes work. I assume that every 'url' here has to be unique? Not sure if this feature should be it's own endpoint and how it should look like. I considered using the `updateTable` and having a query string or something, but I don't know how to do that or if that's the best way to go about it.